File: Server/src/main/java/com/api/server/foodb/Sequence.java
package com.api.server.foodb;

import jakarta.persistence.*;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.Setter;

import java.time.Instant;

@Getter
@Setter
@Entity
@Table(name = "sequences")
public class Sequence {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false)
    private Integer id;

    @Size(max = 255)
    @Column(name = "header")
    private String header;

    @Lob
    @Column(name = "chain")
    private String chain;

    @Column(name = "sequenceable_id")
    private Integer sequenceableId;

    @Size(max = 255)
    @Column(name = "sequenceable_type")
    private String sequenceableType;

    @Size(max = 255)
    @Column(name = "type")
    private String type;

    @Column(name = "created_at")
    private Instant createdAt;

    @Column(name = "updated_at")
    private Instant updatedAt;

}

File: DotNetServer/DotNetServer/Core/Entities/Foodb/Sequence.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace DotNetServer.Core.Entities.Foodb;

[Table("sequences")]
[Index("SequenceableId", Name = "index_sequences_on_sequenceable_id")]
[Index("SequenceableId", "SequenceableType", Name = "index_sequences_on_sequenceable_id_and_sequenceable_type")]
[Index("SequenceableType", Name = "index_sequences_on_sequenceable_type")]
public partial class Sequence
{
    [Key] [Column("id")] public int Id { get; set; }

    [Column("header")] [StringLength(255)] public string? Header { get; set; }

    [Column("chain", TypeName = "mediumtext")]
    public string? Chain { get; set; }

    [Column("sequenceable_id")] public int? SequenceableId { get; set; }

    [Column("sequenceable_type")] public string? SequenceableType { get; set; }

    [Column("type")] [StringLength(255)] public string? Type { get; set; }

    [Column("created_at", TypeName = "datetime")]
    public DateTime? CreatedAt { get; set; }

    [Column("updated_at", TypeName = "datetime")]
    public DateTime? UpdatedAt { get; set; }
}